client/src/api/auth.js

```diff
@@ -0,0 +1,14 @@
+export const signApi = (email, password) => {
```

Comment:
To me it's not so clear what a function called `signApi` does. What does this function do? It logs in a user, so we could name it `loginUser` or `signInUser` to be more easily understood